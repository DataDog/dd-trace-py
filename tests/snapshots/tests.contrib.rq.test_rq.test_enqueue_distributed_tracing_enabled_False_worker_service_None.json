[[
  {
    "name": "rq.worker.perform_job",
    "service": "rq-worker",
    "resource": "tests.contrib.rq.jobs.job_add1",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "worker",
    "meta": {
      "job.id": "cb601426-9ba6-4ff7-baff-4af051664698",
      "job.origin": "q",
      "job.status": "finished",
      "runtime-id": "9f7973a4da1d44a18c0117159fe67295"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "system.pid": 67081
    },
    "duration": 19190000,
    "start": 1632150774362649000
  },
     {
       "name": "rq.job.perform",
       "service": "rq-worker",
       "resource": "tests.contrib.rq.jobs.job_add1",
       "trace_id": 0,
       "span_id": 2,
       "parent_id": 1,
       "meta": {
         "job.id": "cb601426-9ba6-4ff7-baff-4af051664698"
       },
       "duration": 3202000,
       "start": 1632150774369990000
     }],
[
  {
    "name": "rq.queue.enqueue_job",
    "service": "rq",
    "resource": "tests.contrib.rq.jobs.job_add1",
    "trace_id": 1,
    "span_id": 1,
    "parent_id": 0,
    "type": "worker",
    "meta": {
      "job.func_name": "tests.contrib.rq.jobs.job_add1",
      "job.id": "cb601426-9ba6-4ff7-baff-4af051664698",
      "queue.name": "q",
      "runtime-id": "2fae27a28fd644a6bd2fbbd179813bd4"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "system.pid": 67067
    },
    "duration": 6343000,
    "start": 1632150774020518000
  }]]
