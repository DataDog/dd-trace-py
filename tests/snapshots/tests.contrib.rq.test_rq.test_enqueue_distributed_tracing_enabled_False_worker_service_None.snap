[[{"name" "rq.queue.enqueue_job"
   "service" "rq"
   "resource" "tests.contrib.rq.jobs.job_add1"
   "type" "worker"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1631053637662518300
   "duration" 2733000
   "meta" {"runtime-id" "b1fd435dff564be79ebf4876f19e3401"
           "queue.name" "q"
           "job.id" "a15302ae-5332-4986-accf-4e5b8cb766fb"
           "job.func_name" "tests.contrib.rq.jobs.job_add1"}
   "metrics" {"_dd.agent_psr" 1.0
              "_sampling_priority_v1" 1
              "system.pid" 2666
              "_dd.tracer_kr" 1.0}}]
 [{"name" "rq.worker.perform_job"
   "service" "rq-worker"
   "resource" "tests.contrib.rq.jobs.job_add1"
   "type" "worker"
   "error" 0
   "span_id" 0
   "trace_id" 1
   "parent_id" nil
   "start" 1631053640905464600
   "duration" 21301700
   "meta" {"runtime-id" "03a7167e3899435fa18ab9f018314093"
           "job.id" "a15302ae-5332-4986-accf-4e5b8cb766fb"
           "job.status" "finished"
           "job.origin" "q"}
   "metrics" {"_dd.agent_psr" 1.0
              "_sampling_priority_v1" 1
              "system.pid" 2679
              "_dd.tracer_kr" 1.0}}
      {"name" "rq.job.perform"
       "service" "rq-worker"
       "resource" "tests.contrib.rq.jobs.job_add1"
       "error" 0
       "span_id" 1
       "trace_id" 1
       "parent_id" 0
       "start" 1631053640908800000
       "duration" 13139700
       "meta" {"job.id" "a15302ae-5332-4986-accf-4e5b8cb766fb"}}]]
